### PR TITLE
Adds default client pixel offset tracking and has reset_view delay until screen shake completes.

### PR DESCRIPTION
**Pull request by MistakeNot4892**

### CODE REVIEW

**Review by comma — DISMISSED** [review]

File: code/modules/mob/mob.dm [hunk 1]
```diff
@@ -279,22 +279,25 @@
...
-				client.eye = client.mob
-				client.perspective = MOB_PERSPECTIVE
-			else
-				client.perspective = EYE_PERSPECTIVE
-				client.eye = loc
-	return
-
+	set waitfor = 0
+	while(shakecamera && client && !QDELETED(src))
+		sleep(1)
```
Comment (Lohikar) [inline code/modules/mob/mob.dm:284]: Probably should be `stoplag(1)` instead of sleep.

Comment (Lohikar) [reply code/modules/mob/mob.dm:284]: On second thought, these're all nullchecks - no need.

Comment (MistakeNot4892) [reply code/modules/mob/mob.dm:284]: As discussed on Discord, would prefer to leave it as `sleep(1)` for clarity.

**Comment by MistakeNot4892** [comment]
Please review this.

**Review by comma — APPROVED** [review]

**Pull request merged**